backend/fleet_management/models.py

```diff
@@ -41,6 +41,7 @@ class Passenger(models.Model):
     first_name = models.CharField(max_length=60, blank=False)
     last_name = models.CharField(max_length=60, blank=False)
     email = models.EmailField(blank=False)
+    country = CountryField(blank_label='(select country)', null=False, default='')
```

Comment:
How about using NULL for value that is not set?
Setting it to an empty string means that it was set to this value on purpose,

Reply:
Fixed https://github.com/CodeForPoznan/pah-fm/pull/326/commits/bdca70d9425f2b60d40adbf7a4ed251d2363c1a7